src/components/Requests/Requests.tsx

```diff
@@ -0,0 +1,43 @@
+import {Actions} from "store/action";
+import {Request as RequestModel} from "types/request";
+import "./Requests.scss";
+import {useDispatch} from "react-redux";
+import {Participant} from "types/participant";
+import {Request} from "./Request";
+
+export interface RequestsProps {
+  requests: RequestModel[];
+  participantsWithRaisedHand: Participant[];
+}
+
+export const Requests = ({requests, participantsWithRaisedHand}: RequestsProps) => {
+  const dispatch = useDispatch();
+
+  const handleJoin = (user: string, acceptJoin?: boolean) => {
+    if (acceptJoin) {
+      dispatch(Actions.acceptJoinRequests([user]));
+    } else {
+      dispatch(Actions.rejectJoinRequests([user]));
+    }
+  };
+
+  const lowerHand = (user: string) => {
+    dispatch(Actions.setRaisedHand(user, false));
+  };
+
+  return requests.length !== 0 || participantsWithRaisedHand.length !== 0 ? (
```

Comment:
should also work as: `return requests.length || participantsWithRaisedHand.length ? (` because 0 is a falsy value